components/throttle.js

```diff
@@ -39,15 +39,13 @@ module.exports = wrapper(throttler);
  * @return a hash with the next elements to be processed.
  */
 function throttler(throttle_size, file_envvar, encoding) {
-
     var node = this.nodeInstance;
     if (_.isUndefined(node.throttle)) node['throttle'] = {}; 
 
     // Get the number of hash elements being requested
     var throttleSize = parseInt(throttle_size);
```

Comment:
This could also be written as var throttleSize = _.isFinite(throttle_size) ? +throttle_size : 1;